frontend/src/components/common/ReleaseNotes/UpdatePopup.tsx

```diff
@@ -2,11 +2,52 @@ import { Button, Snackbar } from '@material-ui/core';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
-function UpdatePopup(props: { releaseDownloadURL: string }) {
+function UpdatePopup(props: {
+  releaseDownloadURL?: string | null;
+  fetchingRelease?: boolean;
+  releaseFetchFailed?: boolean;
+}) {
   const [show, setShow] = React.useState(true);
-  const { releaseDownloadURL } = props;
+  const [releaseFetchShow, setReleaseFetchShow] = React.useState(true);
+  const { releaseDownloadURL, fetchingRelease, releaseFetchFailed } = props;
   const { t } = useTranslation('frequent');
 
+  if (fetchingRelease) {
+    return (
+      <Snackbar
+        open={releaseFetchShow}
+        message={t('release|fetching release...')}
+        ContentProps={{
+          'aria-describedby': t('release|fetching release message'),
```

Comment:
This property is not supposed to get a description but rather an ID:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

Reply:
updated